airflow/models/xcom.py

```diff
@@ -723,6 +723,20 @@ def __eq__(self, other: Any) -> bool:
             return all(x == y for x, y in z)
         return NotImplemented
 
+    def __getstate__(self) -> Any:
+        # We don't want to get into the trouble to pickle the entire SQLAlchemy
```

Comment:
```suggestion
        # We don't want to get into the trouble to pickle the entire SQLAlchemy
```
it's a little unclear whether you mean it's a big effort to do, or it's a problematic thing to do.

Reply:
Ah, the root issue here is that SQLAlchemy objects don’t pickle well out of the box, so the big effort comes from we need to implement _custom logic_ for every thing in a SQLAlchemy query (e.g. filters, order_bys, hints). I’ll add this to the comments to clarify.

Reply:
got you ... ok then i would just suggest changing to "go _through the trouble of pickling the"

Reply:
wait it's "go _to_ the trouble" :) had to google, myself